src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -328,7 +328,10 @@ namespace WaterToAirHeatPumpSimple {
             state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).RatedWaterVolFlowRate = NumArray(2);
```

Comment:
I'm not going to ask you to do any of this here (unless you really want to), but I am just going to make a few comments because I don't get to comment on your PR's very often and I don't want you to feel neglected. 

References are evil. `auto` is also evil. But there is one use in which both references and `auto` are slightly less evil: the shortcut object reference, e.g., `auto &simpleWAHP = state.dataWatertoAirHeatPumpSimple->SimpleWatertoAirHP(HPNum);`. They make code more readable, and also run faster in debug mode.

Reply:
Thanks for the comments! I'll address the issues you pointed out once the rest of the proposed changes have been reviewed.